tallycash/extension
# 📐 Better network fee estimates

_The problem_
> The below is the difference between the quoted gas cost and actual gas cost
for 3 swaps done on Tally and MM:
Tally MetaMask
$20.33 $4.63
$43.21 $5.27
$18.67 $6.99

– @mr-michael 
<br/>

This PR **gets our gas estimates to striking distance to MM by using the current
base fee** father than the max fee, which can often times be double. Previously,
our calculation for the estimated network fee didn't make much sense (added
priority fee to max fee), so this also cleans that up.

To test:
- Propose a swap in Tally, check estimated network fee
- Propose an essentially identical swap in MM, check estimated network fee
- See if they're near each other

Screenshots:
<table>
<tbody>
<tr>
<td><b>🦊 MetaMask</b></td>
<td><b>🐶 Tally Ho</b></td>
</tr>
<tr>
<td>$24.32</td>
<td>$24.45</td>
</tr>
<tr>
<td><img width="310" alt="Metamask Estimate" src="https://user-images.githubusercontent.com/1918798/169537448-1cc312de-43b0-4366-8964-60ec40e46abd.png"></td>
<td><img width="310" alt="Tally Ho Estimate" src="https://user-images.githubusercontent.com/1918798/169537430-3033df08-204f-46f9-9bcc-5bc2c8e113ae.png"></td>
</tr>
</tbody>
</table>

Closes #1337

🤘
